sdk_container/src/third_party/coreos-overlay/app-admin/etcd-wrapper/etcd-wrapper-3.5.0-r2.ebuild

```diff
@@ -1,25 +1,19 @@
 # Copyright (c) 2016 CoreOS, Inc.. All rights reserved.
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
 
-TMPFILES_OPTIONAL=1
```

Comment:
`pkgcheck` complained about the dependency not being manually added via some USE flag. It's not needed by the ebuild itself, but it installs configuration with `dotmpfiles`, which implies that the dependency is needed at runtime. That's what `TMPFILES_OPTIONAL` controls. I don't know why the line was added here, but it's basically a no-op because the virtual just resolves to `sys-apps/systemd` anyway.